bases/flux-app-v2/customer/kustomization.yaml

```diff
@@ -6,10 +6,18 @@ helmCharts:
     namespace: flux-system
     repo: https://giantswarm.github.io/giantswarm-catalog/
     releaseName: customer-flux
-    version: v1.0.0
+    version: 1.3.1
     valuesInline:
       images:
         registry: gsoci.azurecr.io
+      cilium:
+        enforce: true
+      podMonitors:
+        enabled: false
```

Comment:
What do you think about this one? This technically does not create the metric collectors for the `flux-system` controllers, but do we need them? 🤔 Thinking we are having alerts configured for the GS part only.

Reply:
Hmm, actually there are a few:
- https://github.com/giantswarm/prometheus-rules/blob/3407f9b7f5accafa6f7fd5302c287ba273bbeb09/helm/prometheus-rules/templates/alerting-rules/flux.rules.yml#L151
- https://github.com/giantswarm/prometheus-rules/blob/3407f9b7f5accafa6f7fd5302c287ba273bbeb09/helm/prometheus-rules/templates/alerting-rules/flux.rules.yml#L261

Reply:
Enabled it for customer flux too after all.